test/livebook_web/live/hub_live_test.exs

```diff
@@ -0,0 +1,227 @@
+defmodule LivebookWeb.HubLiveTest do
+  use LivebookWeb.ConnCase, async: true
+
+  import Phoenix.LiveViewTest
+
+  alias Livebook.Hubs
+  alias Livebook.Hubs.Hub
+
+  setup do
+    on_exit(fn ->
+      for %{id: hub_id} <- Hubs.fetch_hubs() do
+        Livebook.Storage.current().delete(:hub, hub_id)
+      end
+    end)
+
+    :ok
+  end
+
+  test "render hub selection cards", %{conn: conn} do
+    {:ok, _view, html} = live(conn, "/hub")
+
+    assert html =~ "Fly"
+    assert html =~ "Livebook Enterprise"
+  end
+
+  describe "fly" do
+    test "renders the second step", %{conn: conn} do
+      {:ok, view, _html} = live(conn, "/hub")
+
+      assert view
+             |> element("#fly")
+             |> render_click() =~ "Deploy notebooks to your Fly account."
+    end
+
+    test "renders filled form with existing hub", %{conn: conn} do
+      hub = %Hub{
+        id: "l3soyvjmvtmwtl6l2drnbfuvltipprge",
+        type: "fly",
+        name: "My Foo",
+        label: "Foo Bar",
+        token: "foo",
+        color: "#FF00FF"
+      }
+
+      Hubs.save_hub(hub)
+
+      {:ok, _view, html} = live(conn, "/hub/" <> hub.id)
+
+      assert html =~ hub.name
+      assert html =~ hub.color
+    end
```

Comment:
For me you can delete both of these tests, because you end-up testing them in the "persists hub" and "updates hub". :)